src/app/map/map.component.html

```diff
@@ -4,7 +4,135 @@
 	<div class="map-parent">
 		<div class="button-container fe1">
 			<div class="button-box">
-				<a class='btn btn--white' style=" background-color: #caf0f8; font-size: 15px;" (click)="this.openDialog()">Nicoles ripple thingy</a>
+				<a class='btn btn--white' style=" background-color: #caf0f8; font-size: 15px;" (click)="this.openDialog()">Mod 1 FrontEnd</a>
```

Comment:
Great job making the ripple effect, we'll need to work on making it conditional but not sure we'll get to it with the time left